docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>Would use <code>tau_a = tau_b = 0.999</code></p><p>I also think being a bit more verbose would be nice here; maybe:</p><p>We will now define an unbalanced OT problem with <code>tau_a = tau_b = 0.999</code> to relax both the source and target marginal constraints.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>